cli/cmd/root.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/oasisprotocol/oasis-sdk/cli/config"
+	_ "github.com/oasisprotocol/oasis-sdk/cli/wallet/file" // Register file wallet backend.
+)
+
+var (
+	cfgFile string
+
+	rootCmd = &cobra.Command{
+		Use:     "oasis",
+		Short:   "CLI for interacting with the Oasis network",
+		Version: "0.1.0",
+	}
+)
+
+// Execute executes the root command.
+func Execute() error {
+	return rootCmd.Execute()
+}
+
+func initConfig() {
+	v := viper.New()
+
+	if cfgFile != "" {
+		// Use config file from the flag.
+		v.SetConfigFile(cfgFile)
+	} else {
+		const configFilename = "config.toml"
+		configDir := config.Directory()
+		configPath := filepath.Join(configDir, configFilename)
+
+		v.AddConfigPath(configDir)
+		v.SetConfigType("toml")
```

Comment:
Do you think TOML will be better for the users :slightly_smiling_face: ?
I'm a bit more in favor or YAML :slightly_smiling_face: